layout/Login/Login.tsx

```diff
@@ -16,12 +16,12 @@ import { LoginForm } from "./components";
 function Login() {
   return (
     <div className="min-h-screen select-none overflow-hidden bg-secondary">
-      <Return componentStyle="sm:ml-20 mt-20 sm:mt-20" />
+      <Return componentStyle="sm:ml-20 mt-20 sm:mt-20 z-10 pl-10" />
```

Comment:
Thank you 🙇! 
Just make sure to remove the left padding when on mobile screens to center the text properly, like this:
```suggestion
      <Return componentStyle="sm:ml-20 mt-20 sm:mt-20 z-10 sm:pl-10" />
```